test/JDBC/upgrade/13_4/schedule

```diff
@@ -234,3 +234,4 @@ babel-4517
 BABEL-4641-before-16_3-or-15_7-or-14_12
 BABEL-4863-before-16_3-or-15_7-or-14_12
 BABEL-4869
+space
```

Comment:
NIT: Add new line at the end of schedule file. Same applicable for all other schedule file as well.

Reply:
Done